docs/development.rst

```diff
@@ -505,463 +505,795 @@ point, with diagrams, and an alternative workflow, see `the git docs
-  (2009) Out of Africa demographic model has the ``id`` "OutOfAfrica_3G09". See
-  :ref:`sec_development_naming_conventions` for more details.
-* ``description``: A brief one-line description of the demographic model.
-* ``long_description``: A longer description (say, a concise paragraph) that describes
-  the model in more detail.
-* ``populations``: A list of ``stdpopsim.Population`` objects, which have their own
-  ``id`` and ``description``. For example, the Thousand Genomes Project Yoruba panel
-  could be defined as ``stdpopsim.Population(id="YRI", description="1000 Genomes YRI
-  (Yorubans)")``.
-* ``citations``: A list of ``stdpopsim.Citation`` objects for the appropriate citation
-  for this model. The citation object requires author, year, and doi information, and
-  a specified reason for citing this model.
-* ``generation_time``: The generation time for the species in years. If you are
-  implementing a generic model, the generation time should default to 1.
-* ``mutation_rate``: The mutation rate assumed during the inference of this demographic
-  model, per bp per generation, if a mutation rate was used. If no mutation
-  rate is associated with this demographic model, which is generally uncommon
-  but possible, depending on the inference method, the mutation rate should be
-  set to ``None``.
-
-Every demographic model has a few necessary features or attributes. First of all,
-demographic models are defined by the population sizes, migration rates, split and
-admixture times, and generation lengths given in the source publication. We often take
-the point estimates for each of the values from the best fit model (for example, the
-parameters that give the maximum likelihood fit), which are translated into
-`msprime`-formatted demographic inputs.
-
-`Msprime`-defined demographic models are specified through the
-``population_configurations``, ``migration_matrix``, and ``demographic_events``. If this
-is your first time specifying a model using `msprime`, it's worth taking some time to
-read through the `msprime`
-`documentation and tutorials <https://tskit.dev/msprime/docs/stable/quickstart.html>`_.
+    _mean_recombination_rate = 200 / 124000 / 2 / 1e6
+    _recombination_rate_data = {str(j): _mean_recombination_rate for j in range(1, 6)}
+    _recombination_rate_data["Mt"] = 0
+    _recombination_rate_data["Pt"] = 0
+
+    _species_ploidy = 2
+    _ploidy = {
+      "1": _species_ploidy,
+      "2": _species_ploidy,
+      "3": _species_ploidy,
+      "4": _species_ploidy,
+      "5": _species_ploidy,
+      "6": _species_ploidy,
+      "Mt": 1,
+      "Pt": 1,
+    }
+
+We then use another object (``_chromosomes``) to wrap up these
+values together with the mutation rate, which is set to the
+genome-wide average value for every chromosome.
+Chromosome ids and lengths are taken from the ``data`` object defined in
+the ``genome_data.py`` file.
+
+.. code-block:: python
 
```

Comment:
I wonder if we should also mention here how to specify bacterial species (with "bacterial recombination"). Gene conversion could/should also be mentioned here. Maybe I should add another header "Recombination models"?